docs/changelog.md

```diff
@@ -5,6 +5,8 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [next]
```

Comment:
There should be no `[next]`, just list items under `HEAD`; please remove.

Reply:
Done.